src/presentation/api/controller/sslController.go

```diff
@@ -33,19 +33,32 @@ func GetSslPairsController(c echo.Context) error {
 	return apiHelper.ResponseWrapper(c, http.StatusOK, sslPairsList)
 }
 
-func parseVirtualHosts(vhosts []interface{}) []valueObject.Fqdn {
-	var virtualHosts []valueObject.Fqdn
-
-	for _, vhost := range vhosts {
-		vhostStr, assertOk := vhost.(string)
-		if !assertOk {
-			panic("InvalidVirtualHosts")
+func parseVirtualHosts(vhostsBodyInput interface{}) []valueObject.Fqdn {
+	var vhosts []valueObject.Fqdn
+
+	switch rawVhosts := vhostsBodyInput.(type) {
+	case string:
+		vhosts = append(
+			vhosts,
+			valueObject.NewFqdnPanic(rawVhosts),
+		)
+	case []interface{}:
+		for _, rawVhostInterface := range rawVhosts {
+			vhostStr, assertOk := rawVhostInterface.(string)
+			if !assertOk {
+				continue
+			}
+
+			vhost, err := valueObject.NewFqdn(vhostStr)
+			if err != nil {
+				continue
+			}
+
+			vhosts = append(vhosts, vhost)
```

Comment:
"case []interface{}" me parece exagerado. A lógica aqui poderia ser mais simples:

1. Se não for  []string{}, verifica se é string e joga a string pra um slice. Se não for string, retorna erro, pois só pode ser []string{} ou string.
2. Converte os itens raw da slice em Fqdn num for.

Reply:
Ajustado.